pkg/controller/launch/pool_manager.go

```diff
@@ -126,7 +128,7 @@ func removeNodePool(pool []models.NodePoolInfo, name string) ([]models.NodePoolI
 }
 
 func (cpm *ConcretePoolManager) SetStatus(status *PoolStatus) error {
-	healthy, schedulable := cpm.healthyAndSchedulable()
+	healthy, schedulable, _ := cpm.healthyAndSchedulable([]string{})
```

Comment:
Why don't we pass `status.Nodes`?

Reply:
The ordered nodes are not used there. I wanted to avoid sorting stuff that is not needed later on. Maybe a bit ambiguous.